apps/AppWithWearable/flavorizr.yaml

```diff
@@ -0,0 +1,17 @@
+flavors:
+
+  development:
+    app:
+      name: "Friend DEV"
+    android:
+      applicationId: "com.friend.ios.dev"
+    ios:
+      bundleId: "com.friend.ios.dev"
+
+  production:
+    app:
+      name: "Friend"
+    android:
+      applicationId: "com.friend.ios"
+    ios:
+      bundleId: "com.friend.ios"
```

Comment:
The bundle ID for iOS is incorrect

Reply:
Also @costa96 could you upload screenshot of having both apps running locally? thanks

I'm not sure who worked on this @costa96 or @Hyla96

Reply:
For android and iOS

Reply:
Sorry, forgot to commit the changes with the right git account. It doesn't really matter, as you can guess I'm using two accounts. Regarding the bundle id, as far as I can see that's how the project is configured.
![Screenshot 2024-06-19 at 07 28 54](https://github.com/BasedHardware/Friend/assets/80916787/ea89d25a-2ad2-41db-970d-9f5bf372e4b7)